reference/release-notes/1.25.1.md

```diff
@@ -0,0 +1,85 @@
+---
+orphan: true
+---
+# 1.25.1
```

Comment:
Can you add a known issue here which says that the images don't bump the security level for the `ro.build.version.security_patch` and `ro.vendor.build.security_patch` properties to `2025-03-05` due to an internal mistake but all patches as listed below have been imported and are part of the images released with 1.25.1?